tooling/Microsoft.MonoDevelop.RazorAddin/Microsoft.MonoDevelop.RazorAddin.csproj

```diff
@@ -0,0 +1,72 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="AddinMetadata.props" />
+  
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+    <EnableApiCheck>false</EnableApiCheck>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="Properties\_Manifest.addin.xml" LogicalName="_Manifest.addin.xml" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!-- 
+      By default the MonoDevelop.Addins package has restore tasks that depend on monodevelop existing on the machine. 
+      We can avoid this requirement by not letting it auto-restore monodevelop dependencies because we know which ones
+      we depend on and can bring them in manually.
+    -->
+    <PackageReference Include="MonoDevelop.Addins" Version="$(MonoDevelopAddinsVersion)" NoWarn="KRB4002" ExcludeAssets="build" />
```

Comment:
Should this go in the lineup?

Reply:
It could but I'd still need to have the version so I can HintPath below. Going to hold off on having this in the LineUp until another project needs it.